test/components/time_series/database/test_database.py

```diff
@@ -1,266 +1,203 @@
 """
-Tests for database services
-Run with: uv run development/test_database.py
+Tests for database services using pytest and SQLite
+Run with: uv run pytest test/components/time_series/database/test_database.py
```

Comment:
Maybe put this into the `README.md`? YOu can also run it with `poe test -k test_datebase.py` or `pytest test -k test_database.py` directly.